src/main/jbake/content/documentation/bundles/org-apache-sling-junit-bundles.md

```diff
@@ -27,6 +27,42 @@ the test class names, like for example:
 
     Sling-Test-Regexp=com.example.*ServerSideTest
 
+### JUnit 4 Support
+All that is required is the installation of the the Apache Sling JUnit Core bundle. The bundle exports the packages `junit.*`, `org.junit.*` and additionally `org.hamcrest.*`. Note however that the `org.junit.platform.*` packages, which contain the generic testing platform developed for JUnit 5, are NOT exported.  
+
+### JUnit 5 Support (since version 2.0.0)
```

Comment:
version 1.1.0?

Reply:
Fixed this after release before I saw your comment ;)